core/services/offchainreporting/OCRContract.go

```diff
@@ -0,0 +1,228 @@
+package offchainreporting
+
+import (
+	"context"
+	"math/big"
+	"strings"
+	"sync"
+	"time"
+
+	ethereum "github.com/ethereum/go-ethereum"
+	gethCommon "github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/core/services/eth"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"github.com/smartcontractkit/chainlink/offchainreporting/confighelper"
+	"github.com/smartcontractkit/chainlink/offchainreporting/gethwrappers/offchainaggregator"
+	ocrtypes "github.com/smartcontractkit/chainlink/offchainreporting/types"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/pkg/errors"
+)
+
+var (
+	OCRContractConfigSet = getConfigSetHash()
+)
+
+type (
+	LogConfigSet struct {
+		eth.GethRawLog
+		ConfigCount  uint64
+		Oracles      []gethCommon.Address
```

Comment:
```suggestion
		Signers      []gethCommon.Address
```

Reply:
Again Im confused as to how this got to be wrong... and if it is wrong, how its working at all